tsconfig.defaults.json

```diff
@@ -1,8 +1,8 @@
 {
   "compilerOptions": {
     "incremental": true,
-    "target": "es2020",
-    "module": "ES2020",
+    "target": "ES2022",
+    "module": "ES2022",
```

Comment:
any reason you changed these? if you do should also change target in SAMtemplates/functions/main.yaml and SAMtemplates/functions/main_template.old.yaml

Reply:
Turned out to be unnecessary, as no internal module mocking was needed. Have reverted.